src/utils/dom-to-image.js

```diff
@@ -470,7 +476,16 @@ function newUtil() {
       image.onload = () => {
         resolve(image);
       };
-      image.onerror = reject;
+      image.onerror = (err) => {
+        const message = `[kepler.gl] Failed to create image from data uri.
+        Copy the uri in the javascript console when reporting this bug.
+        The uri is the string starts with "data:image/png"`;
+        console.warn(message, err);
+        console.log(uri);
```

Comment:
this?

Reply:
print the uri in console, so people can paste it while reporting this bug